### DISCUSSION TITLE
No header file called spix_export.h in this repo

### DISCUSSION THREAD

**Discussion by 02deno** [faaxm/spix#85] [General]
There is no header file called "spix_export.h" in lib/include/Spix but header files like AnyRpcServer.h includes this file and I get an error because of this. Where is this missing file? 
![image](https://user-images.githubusercontent.com/103145955/223102428-d55cda10-4724-424a-a401-57f7c203763b.png)

**Comment by faaxm** [comment]
It gets generated by cmake here:
https://github.com/faaxm/spix/blob/4de44072d76b49ceaae9c57046f2260948c065a4/lib/CMakeLists.txt#L141-L144

**Reply by 02deno** [reply]
Thank you very much :))
